pangeo_forge/pipelines/base.py

```diff
@@ -119,3 +120,23 @@ def storage(self) -> Storage:
             and ``self.path`` as the path.
         """
         return GitHub(self.repo, path=self.path)
+
+    def _generate_run(self, source):
+        name = type(self).__name__
+        with open(source, encoding="utf-8") as f:
+            source = f.read()
+
+        return textwrap.dedent(
+            """\
+            # file: run.py
+            # generated by pangeo-forge.
+
+            {source}
+
+            # ----------------------------------------------------------------
+            pipe = {name}()
+            flow = pipe.flow
+            flow.storage = pipe.storage
+            flow.environment = pipe.environment
+            """
+        ).format(source=source, name=name)
```

Comment:
This feels a bit hacky, but I understand why it is needed. Hopefully upstream changes in prefect could eventually make it unnecessary.